resources/careconnect-to-ukcore/medicationstatement/input/MedicationStatementDosageLastChanged-Extension-3to4_000.json

```diff
@@ -0,0 +1 @@
+{"resourceType":"MedicationStatement","extension":[{"url":"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-MedicationDosageLastChanged-1","valueDateTime":"2020-01-28"}]}
```

Comment:
@declankieran-nhsd input files need formatting. They are not consistent with others.

Reply:
I pushed a fix.